update expo docs (#1666)
clerk/clerk-docs: docs/_partials/expo/oauth-custom-flow.mdx

> The following example demonstrates a component that creates a custom OAuth sign-in flow for [Google accounts](/docs/authentication/social-connections/google).

Author: ```suggestion
The following example demonstrates how to create a custom OAuth sign-in flow for [Google accounts](/docs/authentication/social-connections/google).
```
